controllers/specialresource.go

```diff
@@ -21,11 +21,6 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
-// GetName of the special resource operator
```

Comment:
i am probably missing something, but where is the new implementation of this function

Reply:
There is no new implementation of that function. It was only used to name the `ClusterOperator` resource; it is now the `operatorName` field of the `clusterOperatorManager` struct.